packages/manager/modules/pci/src/projects/project/kubernetes/details/node-pool/node-pool.html

```diff
@@ -70,6 +70,25 @@
             >
             </span>
         </oui-datagrid-column>
+        <oui-datagrid-column
+            data-title=":: 'kube_node_pool_anti_affinity' | translate"
+            data-property="antiAffinity"
+            data-type="boolean"
+            data-sortable
+        >
+            <div
+                data-ng-if="$row.antiAffinity"
+                class="d-flex justify-content-center"
+            >
+                <i class="oui-icon oui-icon-check" aria-hidden="true"></i>
```

Comment:
```suggestion
                <span class="oui-icon oui-icon-check" aria-hidden="true"></span>
```
Also missing a `span` to describe the value